sdk/servicebus/azure-servicebus/tests/mgmt_tests/test_mgmt_queues.py

```diff
@@ -255,6 +256,24 @@ def test_mgmt_queue_create_with_queue_description(self, servicebus_namespace_con
             #requires_duplicate_detection=True,
             requires_session=True
         )
+
+        mgmt_service.create_queue(
+            queue_name_2,
+            auto_delete_on_idle="PT10M1S",
+            dead_lettering_on_message_expiration=True,
+            default_message_time_to_live="PT11M32S",
+            duplicate_detection_history_time_window="PT12M3S",
+            enable_batched_operations=True,
+            enable_express=True,
+            enable_partitioning=True,
+            forward_dead_lettered_messages_to=topic_name,
+            forward_to=topic_name,
+            lock_duration="PT13S",
```

Comment:
maybe you can also test updating a few of these using string format, like in [`test_mgmt_queue_update_success`](https://github.com/Azure/azure-sdk-for-python/blob/7fe58e95f2ef794aee172ba81f58fdc783c650bd/sdk/servicebus/azure-servicebus/tests/mgmt_tests/test_mgmt_queues.py#L299)? (and mgmt_topic, mgmt_subscription)

Reply:
good catch, I actually found that there's a bug in the `_model_workaround.py.avoid_timedelta_overflow` method which disallows passing a timespan string (throwing AttributeError as the method always expect a timedelta object) for update.

code updated!